tests/cupy_tests/math_tests/test_window.py

```diff
@@ -0,0 +1,44 @@
+import unittest
+
+from cupy import testing
+
+
+@testing.gpu
+class TestWindow(unittest.TestCase):
```

Comment:
Use parameterized test. For example https://github.com/cupy/cupy/blob/master/tests/cupy_tests/creation_tests/test_from_data.py#L170